docs/framework/whats-new/index.md

```diff
@@ -36,23 +37,381 @@ This article does not provide comprehensive information about each new feature a
 
-<a name="v471"></a> 
-## Introducing the .NET Framework 4.7.1
+<a name="v472"></a> 
+## Introducing the .NET Framework 4.7.2
+
+The .NET Framework 4.7.2 builds on previous versions of the .NET Framework 4.x by adding many new fixes and several new features while remaining a very stable product.
+
+### Downloading and installing the .NET Framework 4.7.2
+ 
+You can download the .NET Framework 4.7.2  from the following locations:
+
+- [.NET Framework 4.7.2 Web Installer](http://go.microsoft.com/fwlink/?LinkId=863262)
+
+- [NET Framework 4.7.2 Offline Installer](http://go.microsoft.com/fwlink/?LinkId=863258)
+
+The .NET Framework 4.7.2 can be installed on Windows 10, Windows 8.1, Windows 7 SP1, and the corresponding server platforms starting with Windows Server 2008 R2 SP1. You can install the .NET Framework 4.7.2 by using either the web installer or the offline installer. The recommended way for most users is to use the web installer.
+
+You can target the .NET Framework 4.7.2 in Visual Studio 2012 or later by installing the [.NET Framework 4.7.2 Developer Pack](http://go.microsoft.com/fwlink/?LinkId=863261). 
+
+### What's new in the .NET Framework 4.7.2
+
+The .NET Framework 4.7.2 includes new features in the following areas:
+
+- [Core](#core472)
+- [ASP.NET](#asp-net472)
+- [Networking](net472)
+- [SQL](#sql472)
+- [WPF](#wpf472)
+- [ClickOnce](#ClickOnce472)
+
+A continuing focus in the .NET Framework 4.7.2 is improved accessibility, which allows an application to provide an appropriate experience for users of Assistive Technology. For information on accessibility improvement in the .NET Framework 4.7.2, see [What's new in accessibility in the .NET Framework](whats-new-in-accessibility.md). 
+
+<a name="core-472" />
+#### Core
+
+The .NET Famework 4.7.2 features a large number of cryptographic enhancements, better decompression support for ZIP archives, and additional collection APIs.
+
+**New overloads of RSA.Create and DSA.Create**
+
+The <xref:System.Security.Cryptography.DSA.Create(System.Security.Cryptography.DSAParamteters)?displayProperty=nameWithType> and <xref:System.Security.Cryptography.RSA.Create(System.Security.Cryptography.RSAParamteters)?displayProperty=nameWithType> methods let you supply key parameters when instantiated a new <xref:System.Security.Cryptography.DSA> or <xref:System.Security.Cryptography.RSA> key. They allow you to replace code like the following:
+
+```csharp
+// Before .NET Framework 4.7.2
+using (RSA rsa = RSA.Create())
+{
+   rsa.ImportParameters(rsaParameters);
+   // Other code to execute using the rsa instance.
+}
+``` 
+```vb
```

Comment:
nit: it's good to separate code blocks from text by using new lines. It renders fine but it's easier to read here.